elastic/kibana
# Disable CCR app if license does not allow it

This PR disables (removes from the UI) the Cross Cluster Replication app when the license is either not available or not active.

Fixes https://github.com/elastic/kibana/issues/35664

### How to test

1) Launch ES with the trial on: `yarn es snapshot --license trial`
2) Open the management section and make sure the cross-cluster replication app is in the list

<img width="308" alt="Screen Shot 2019-09-04 at 14 31 02" src="https://user-images.githubusercontent.com/2854616/64255109-0538f500-cf21-11e9-8b53-54837d25fa51.png">

3) Go to "License management" and click on "Revert to basic".
4) After the pages refresh, the cross-cluster replication should not be in the list anymore

<img width="264" alt="Screen Shot 2019-09-03 at 16 45 31" src="https://user-images.githubusercontent.com/2854616/64187618-3bb73700-ce71-11e9-8d93-510bc800cb51.png">
